src/helpers/object.js

```diff
@@ -280,11 +305,16 @@ export function deepObjectSize(object) {
   if (!isObject(object)) {
     return 0;
   }
+
   const recursObjLen = function(obj) {
     let result = 0;
 
     if (isObject(obj)) {
       objectEach(obj, (key) => {
+        if (key === '__children') {
```

Comment:
I get where you're coming from, but I'm afraid until nested rows use a key in the dataset, it should be ignored everywhere else, including the schema, so it's safer for it to be here. What do you think, @wojciechczerniak?

Reply:
I have developed a nested rows blindness 🙄 If you see a better place to handle this then we can do it. Otherwise, we can move forward until the nested rows refactor.

Reply:
You may just extend the function for an extra, generic, argument.